tools/fs/files.ts

```diff
@@ -1625,8 +1625,10 @@ function wrapFsFunc<TArgs extends any[], TResult>(
         }
       }
     }
-
-    const result = fn.apply(fs, args);
+    let result;
+    Profile.time(`files.${fnName} - uncached`, () => {
+      result = fn.apply(fs, args);
+    })
```

Comment:
It is sometimes useful to see how many cache misses there are, but since this adds additional overhead I can remove it.

Reply:
Or behind an envar? We already have some similar ones like the `METEOR_TEST_PRINT_LINKER_CACHE_DEBUG` one

Reply:
That is an option, though the question is if anyone would use it. If not, I could just add it in locally whenever I am profiling something where it matters.

Reply:
True, I honestly had to search for this option because I knew it was there but I'd be very surprised if it's ever used. An abundance of options is not much use if it's not documented, let alone if it's never used.